modules/system/controllers/requestlogs/config_list.yaml

```diff
@@ -8,7 +8,7 @@ modelClass: System\Models\RequestLog
 recordUrl: system/requestlogs/preview/:id
 noRecordsMessage: backend::lang.list.no_records
 recordsPerPage: 30
-showSetup: true
+showSetup: false
```

Comment:
Better mobile styles is not a good enough reason to prevent people from utilizing the showSetup feature to customize their backend lists.